src/model.js

```diff
@@ -355,12 +382,12 @@ module.exports = function(koop) {
         return (180/Math.PI*Math.atan(0.5*(Math.exp(n)-Math.exp(-n))));
     }
 
-    function tileIndexToBBox(x,y,z){
+    function tileIndexToBBox(x,y,z,buffer=0){
         const xmin = tile2long(x, z);
         const xmax = tile2long(x+1, z);
         const ymin = tile2lat(y+1, z);
         const ymax = tile2lat(y, z);
-        return {xmin:xmin, ymin:ymin, xmax:xmax, ymax:ymax};
+        return {xmin:xmin-Math.abs(xmin*buffer), ymin:ymin-Math.abs(ymin*buffer), xmax:xmax+Math.abs(xmax*buffer), ymax:ymax+Math.abs(ymax*buffer)};
```

Comment:
Can you move the math to the lines above?  Then all the calculations are in one spot and you can simply return:

```js
return { xmin, ymin, xmax, ymax }
```

Reply:
Done.